SQUIR-296 Support for Postal code of DK and LU

jignesh1995 opened the pull request:
Adding support for Denmark and Luxembourg postal code in the format:
- DK2000
- DK-2000
- L1222
etc.

Input formats of Denmark Postal Code:
https://en.wikipedia.org/wiki/Postal_codes_in_Denmark

ozankaya commented on formats/LU.json:
```diff
@@ -0,0 +1,23 @@
+{
+  "Description": "DK",
```
Description should be **LU**

jignesh1995 replied:
Updated.

jignesh1995 commented:
> Can you please also run **npm run test**. It should point you to add the new formats to **formats-web.js** file. I updated the travis.yml file to execute all the unit tests. It should now fail if you rebase to mainline.

I did a rebase but found that `npm run test` does not throw any error. Also, as suggested I have added DK and LU to `format-web.json`
